src/index.html

```diff
@@ -5,9 +5,51 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="card" data-qa="card">
+      <div class="card__img">
+        <img src="images/imac.jpeg" alt="icon">
+      </div>
+      <div class="card__description">
+        <p class="card__description-p1">
```

Comment:
I think it is title, better change p to h2, and 
```suggestion
        <p class="card__title">
```